wgpu-core/src/validation.rs

```diff
@@ -729,32 +729,71 @@ fn map_texture_format(format: wgt::TextureFormat) -> naga::TypeInner {
         Tf::Depth32Float | Tf::Depth24Plus | Tf::Depth24PlusStencil8 => {
             panic!("Unexpected depth format")
         }
+
+        // Rgba Compressed Textures
```

Comment:
would be great to have my stuff landing first, since there is a conflict in this bit of code

Reply:
Sounds good